docs/debian-conf.d/auth/01_exim4-config_vexim

```diff
@@ -0,0 +1,24 @@
+
+### 01_exim4-config_vexim
+#########################
+
+plain_login:
```

Comment:
Can you rename these to `plain_vexim` and `login_vexim`?
I'm guessing that these names must be unique, is that correct? In that case, having `vexim` in them seems a natural choice.